shame_script.py

```diff
@@ -1,14 +1,19 @@
 import configparser
+import logging
 import os
 from datetime import datetime, time
-from typing import List, Union
-
+from typing import List, TypedDict
 
+import aiohttp
 import discord
-import requests
 from discord.ext import commands, tasks
+from table2ascii import Alignment, TableStyle, table2ascii
+
 from discord_signup import signup
-from table2ascii import table2ascii, TableStyle, Alignment
+from log_setup import trace_config
```

Comment:
turns out imports have an order they are supposed to be in: three blocks 
1. built ins
2. 3rd party
3. local modules

and then sorted alpha within the blocks